### DISCUSSION TITLE
overlay a blurred with an unblurred version of an input image (= PSD layer)

### DISCUSSION THREAD

**Discussion by Lebostein** [ImageMagick/ImageMagick#4281] [Help]
Hi,

I have a PSD file as source. I convert a single layer from the PSD to a blurred PNG:
`magick convert -blur 0x4 input.psd[2] output.png`

It is possible to add the same layer again (but now unmodified) to the output to generate that specific glow effect I want (see example image)?

Thanks!

![question](https://user-images.githubusercontent.com/23153870/134870705-fd1c5109-5b47-41ae-8fd3-1366f9642d8a.png)

**Comment by snibgo** [comment]
What version of IM, on what platform?

I assume input.psd[2] has transparency.

I suggest you use "magick" only, not "magick convert".

With "magick" only, the order is important. Read the input, then blur it, then write the output.

To composite the blurred verson of the original:
```
magick input.psd[2] ( +clone -blur 0x4 ) -compose DstOver -composite output.png
```
That will work in Windows. For bash, parentheses are special characters and need to be quoted or escaped:
```
magick input.psd[2] \( +clone -blur 0x4 \) -compose DstOver -composite output.png
```
